src/doccano_utils.py

```diff
@@ -0,0 +1,87 @@
+import os
+from doccano_client import DoccanoClient
+
+class DoccanoSession:
+    def __init__(self, server=None):
+        self.username = os.getenv("DOCCANO_USERNAME")
+        self.password = os.getenv("DOCCANO_PASSWORD")
+        self.server = server or 'http://localhost:8000'
+        self.client = self.create_session()
+        self.user = None
+
+    def create_session(self):
+        """
+        Connect and log on to a Doccano server
+        """
+        client = DoccanoClient(self.server)
+        client.login(username=self.username, password=self.password)
+        self.user = client.get_profile()
+        return client
+
+    def create_project(self, name, project_type, description, guideline):
+        """
+        Register a new Doccano project
+        """
+        try:
+            project = self.client.create_project(
+                name=name,
+                project_type=project_type,
+                description=description,
+                guideline=guideline
+            )
+            self.current_project_id = project.id
+            return project
+        except Exception as e:
+            print(f"Failed to create project: {str(e)}")
```

Comment:
I know this is really tempting to do, [but it's generally more harmful than not](https://realpython.com/the-most-diabolical-python-antipattern/)

Unless the library is going to handle a particular `Exception` then this is just obfuscating what is going on as the script won't die at this point - but will try to continue in a state where the project is broken. If the project cannot be created then we probably want the script to halt here.

If you wanted to do this for logging persons I'd log it or `print` and then re-raise the exception [like this](https://www.geeksforgeeks.org/python-reraise-the-last-exception-and-issue-warning/)

```suggestion
        project = self.client.create_project(
            name=name,
            project_type=project_type,
            description=description,
            guideline=guideline
        )
        self.current_project_id = project.id
        return project
```